cfgov/unprocessed/css/molecules/global-search.less

```diff
@@ -111,47 +111,47 @@
       font-size: 18px;
     });
 
-  &:focus {
-    outline: 1px dotted @black;
-  }
+    &:focus {
+      outline: 1px dotted @black;
+    }
 
-  &-open-label {
-    vertical-align: text-top;
-  }
+    &-open-label {
+      vertical-align: text-top;
+    }
 
-  &-close-label {
-    display: none;
-  }
+    &-close-label {
+      display: none;
+    }
 
-  // Hover state for desktop.
-  &:hover {
-    color: @gray-90;
+    // Hover state for desktop.
+    &:hover {
+      color: @gray-90;
+    }
   }
-}
 
-&_content {
-  position: absolute;
-  left: 0;
-
-  &-form {
+  &_content {
     position: absolute;
-    width: 100%;
-  }
+    left: 0;
 
-  &.u-invisible {
-    overflow-x: hidden;
-  }
+    &-form {
+      position: absolute;
+      width: 100%;
+    }
 
-  &[aria-expanded="true"] &-form {
-    display: block;
-  }
+    &.u-invisible {
+      overflow-x: hidden;
```

Comment:
Do we want to be adding to the u-invisible class or should a new utility be defined?

Reply:
Hmmmm I think I mistakenly linted this file and added it to this PR. Its contents haven't been modified (as can be seen when viewing the file with [whitespace changes ignored](https://github.com/cfpb/consumerfinance.gov/pull/7268/files?w=1#diff-2432c2598b33a417af8c309c27551394bbdecd17cfe28ea9960b3cfbf3071c8b)).